src/main/java/com/ishland/c2me/common/optimization/chunkserialization/UnsafeUtil.java

```diff
@@ -0,0 +1,26 @@
+package com.ishland.c2me.common.optimization.chunkserialization;
+
+import com.ishland.c2me.C2MEMod;
+import sun.misc.Unsafe;
+
+import java.lang.reflect.Field;
+
+public class UnsafeUtil {
+    private static final Unsafe INSTANCE;
+
+    static {
+        try {
+            Field field = Unsafe.class.getDeclaredField("theUnsafe");
```

Comment:
This won't work on some JVMs such as Dalvik and potentially oj9. while niche will save future bug reports of people running on exotic JVMs.  see here for a method that should work on pretty much every jvm: https://github.com/duplexsystem/JDKUtils/blob/8690e57682b8e09d3293775d2caf03f01df60de6/src/io/github/duplexsystem/JDKUtils/UnsafeUtils.java#L12